src/apps/ENGINE/ifs.cpp

```diff
@@ -432,9 +432,9 @@ void IFS::Format(char *file_data, long file_size)
         }
     }
 
-    for (n = 0; n < file_size; n++)
+    for (n = -1; n < file_size; n++)
```

Comment:
This fix is in a separate PR.

Reply:
> Thanks for the PR!
> 
> Please remove the ifs.cpp fix from this PR and try to add sdl2 via conan or as a submodule in the worst case.

I found way to get sdl2 via conan, but I have another question: you would like to link sdl2 statically or dynamically?